### DISCUSSION TITLE
Moving activities from one account to another.

### DISCUSSION THREAD

**Discussion by Pindol83** [ghostfolio/ghostfolio#2720] [Ideas]
Hi guys, yesterday by mistake. I imported transactions from one broker into another account, but I couldn't find an easy way to select multiple transactions and move them to a different account. It would be a very useful feature to implement, especially for those who use tools to automatically import transactions from various brokers.

Thanks

**Comment by dtslvr** [comment]
Hi @Pindol83

Here is a similar discussion: https://github.com/ghostfolio/ghostfolio/discussions/1899

**Reply by Pindol83** [reply]
Understood, have you ever thought alternatively about adding an option to allow the deletion or modification of multiple activities at the same time? This way, I could have selected all the activities I wanted to delete and reimport them into the correct account, or even better, modify them as a group and transfer them from one account to another without having to do it one by one.

**Reply by dtslvr** [reply]
Yes, bulk updates would indeed be a valuable addition. Are you interested in taking on this enhancement? Let me know and we can discuss further!

**Reply by Pindol83** [reply]
Thank you for your trust, but I'm not a programmer, so I really wouldn't know how to help you implement it. Let me know if you find someone who can assist you.
